tests/features/tmgmt/tmgmt_poetry.feature

```diff
@@ -17,6 +17,26 @@ Feature: TMGMT Poetry features
       | de        |
       | it        |
 
+  Scenario Outline: Checking automatic counter request.
+    Given I am logged in as a user with the 'administrator' role
+    And I go to "node/add/page"
+    And I fill in "Title" with "<title>"
+    And I fill in "Body" with "<body>"
+    And I press "Save"
+    And I select "Published" from "state"
+    And I press "Apply"
+    Then I click "Translate" in the "primary_tabs" region
+    And I select the radio button "" with the id "edit-languages-fr"
+    And I press "Request translation"
+    And I press "Submit to translator"
+    And I store the job reference of the translation request page
+    Then the poetry translation service received the translation request
```

Comment:
My understanding is that the request happens in that sequence: (generateRequestId)
- IF You find a numero in the db poetry_map: use the numero only if part is < than 99
- if not (part is 99 or you never had a numero) : request a numero
{code}
// If we have found nothing, request a new numero.
    $id_data = array(
      'codeDemandeur' => $settings['code'],
      'annee' => date("Y"),
      'numero' => $settings['counter'],
      'version' => 00,
      'partie' => 00,
      'produit' => 'TRA',
   );
{code}
Then with this request, you now have a numero and can switch to step 1